lib/fencing/st_client.c

```diff
@@ -926,9 +926,11 @@ invoke_registered_callbacks(stonith_t *stonith, xmlNode *msg, int call_id)
                                      cb_info->user_data, cb_info->callback);
 
     } else if ((private->op_callback == NULL) && !pcmk__result_ok(&result)) {
-        crm_warn("Fencing action without registered callback failed: %d (%s)",
+        crm_warn("Fencing action without registered callback failed: %d (%s%s%s)",
                  result.exit_status,
-                 pcmk_exec_status_str(result.execution_status));
+                 pcmk_exec_status_str(result.execution_status),
+                 ((result.exit_reason == NULL)? "" : ": "),
+                 ((result.exit_reason == NULL)? "" : result.exit_reason));
```

Comment:
Not saying anything should be changed here or in other places just noticed that 
we have code using the exit-reason as some kind of amendment if available and
other that replaces status-string-conversion if there is an exit-reason.
Just building upon exit-reason is definitely better for the effort having more concise / more 
natural language errors. We just have to be careful that what we output stays descriptive enough -
like giving a detail without what actually happened. And we are making output harder to parse
automatically. No idea if there is any value in thinking about switches for natural (or even
native) language.